libs/barista-components/copy-to-clipboard/src/copy-to-clipboard.ts

```diff
@@ -53,7 +54,10 @@ const DT_COPY_TO_CLIPBOARD_SUCCESSFUL = 'dt-copy-to-clipboard-successful';
   encapsulation: ViewEncapsulation.Emulated,
 })
 export class DtCopyToClipboard implements AfterContentInit, OnDestroy {
-  constructor(private _cd: ChangeDetectorRef) {}
+  constructor(
+    private _cd: ChangeDetectorRef,
```

Comment:
```suggestion
    private _changeDetectorRef: ChangeDetectorRef,
```

I know that you did not change this one, but as long as we are touching this file, I feel like we could align this one to our coding guidelines.